ci/install.sh

```diff
@@ -14,9 +14,9 @@ fi
 
 # Use default CC to avoid compilation problems when installing Python modules.
 echo "Install neovim module for Python 3."
-CC=cc python3 -m pip -q install --upgrade pynvim
+CC=cc python3 -m pip -q install --user --upgrade pynvim
 echo "Install neovim module for Python 2."
-CC=cc python2 -m pip -q install --upgrade pynvim
+CC=cc python2 -m pip -q install --user --upgrade pynvim
```

Comment:
IIRC using `--user` might end up with binaries not being available in the `$PATH` - but should be fine for the lib.

Reply:
There were [permission issues](https://travis-ci.com/github/jamessan/neovim/jobs/299690353#L463) when not using `--user`.  Would it be preferable to use `sudo` here instead?

Reply:
No, I prefer `--user` over `sudo` if possible.